setup.py

```diff
@@ -30,12 +34,16 @@ def get_version():
     'pytest-asyncio>=0.5.0',
     'pytest-cov>=2.3.1',
     'logbook>=1.0.0',
+    'flake8>=3.0.4',
+    'isort>=4.2.5',
+    'collective.checkdocs>=0.2',
+    'Pygments>=2.1.3'  # required by checkdocs
```

Comment:
What about max versions? Usually compatibility breaks with the next major version.